src/Redux/homepage/homepage.js

```diff
@@ -0,0 +1,100 @@
+import axios from 'axios';
+
+const FETCH_FORECAST_REQUEST = 'homepageStore/homepage/fetch_request';
+const FETCH_FORECAST_SUCCESS = 'homepageStore/homepage/fetch_success';
+const FETCH_FORECAST_FAILURE = 'homepageStore/homepage/fetch_failure';
+const TEST = 'homepageStore/homepage/test';
+
+const FORECAST_URL = 'https://api.open-meteo.com/v1/forecast?';
+const initialiState = {
+  loading: false,
+  error: '',
+  data: [{
+    name: 'Beirut',
+    time: '00:00',
+    temperature: 22,
+  }],
+};
+
+export const test = (payload) => ({
+  type: TEST,
+  payload,
+});
+
+export const fetchForecastRequest = () => ({
+  type: FETCH_FORECAST_REQUEST,
+});
+
+export const fetchForecastSuccess = (payload) => ({
+  type: FETCH_FORECAST_SUCCESS,
+  payload,
+});
+
+export const fetchForecastFailure = () => ({
+  type: FETCH_FORECAST_FAILURE,
+});
+
+export const loopFetchSixCities = () => (dispatch) => {
+  dispatch(fetchForecastRequest);
+  const cities = ['London', 'Amsterdam', 'Paris', 'New York', 'Tokyo', 'Cairo'];
+  const lat = [51.5072, 52.3676, 48.8566, 40.7128, 35.6762, 30.0444];
+  const lng = [0.1276, 4.9041, 2.3522, 74.0060, 139.6503, 31.2357];
+  const arr = [];
+
+  for (let i = 0; i < lat.length; i += 1) {
+    axios.get(`${FORECAST_URL}latitude=${lat[i]}&longitude=${lng[i]}&hourly=temperature_2m`, { headers: {} })
+      .then((response) => {
+        const { data } = response;
+        const { hourly } = data;
+        /* eslint-disable-next-line */
```

Comment:
- [ ] Please remove this eslint diable line its best to fix the issue in the code.